src/libs/actions/connections/index.ts

```diff
@@ -383,7 +384,8 @@ function getSynchronizationErrorMessage(policy: OnyxEntry<Policy>, connectionNam
     }
 
     const connection = policy?.connections?.[connectionName];
-    if (isSyncInProgress || isEmptyObject(connection?.lastSync) || connection?.lastSync?.isSuccessful) {
+
+    if (isSyncInProgress || isEmptyObject(connection?.lastSync) || connection?.lastSync?.isSuccessful !== false || !connection?.lastSync?.errorDate) {
```

Comment:
the issue seems to be fixed only because of this line change, i wonder if we should undo the other changes.
@arosiclair @hungvu193

https://github.com/user-attachments/assets/34e32033-2e25-4b80-9620-b762425ed7da

Reply:
I didn't see that, thanks @hungvu193

Reply:
>If so, we need to use isConnectionInProgress to keep the RBR up to date between Indicator, WorkspaceListPage, PolicyAccountingPage while syncing. 

@hungvu193 should we add this case to the test steps?

Reply:
I think that's ok. Let me update it.

Reply:
Updated